runtimes/python-3.10/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  open-runtimes-python:
+    container_name: open-runtimes-python
```

Comment:
Do we need this file ? Didn't we remove this for the node example ?

Reply:
We did not remove it from node, we removed it from root directory. This one is running example file, the one we removed was running tests.